packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1222,6 +1229,12 @@ class CrudStoreImpl
       return;
     }
 
+    await this.recentQueriesStorage.saveQuery({
+      _ns: this.state.ns,
+      filter,
+      update,
+    });
```

Comment:
This doesn't work the same as recent queries, always saving the query with the same shape as the new item in the store, in comparison to current recents behavior where saving the same query [only updates the last used timestamp](https://github.com/mongodb-js/compass/blob/a2bd0bacdbc2c1b2df6a8954cff7cc30230e5902/packages/compass-query-bar/src/stores/query-bar-reducer.ts#L381-L403). Are we ignoring this behavior on purpose?

Reply:
I did it on purpose because update queries have side effects, so two updates do not have the same result and it IMO makes more sense to have two different history items. This is not defined from a product perspective, so this seems to be the safest bet in terms of not losing information / convenience, as you can't run the same update query twice, unless:

* You write the same query again.
* You open a recent/favorite item.

Reply:
To clarify, you can't run it twice because the modal closes, so you need to rewrite it again, open the recent.

Reply:
I'm not sure why you'd want your recents list to be polluted with the same query showing over and over again (especially as it will start deleting the the oldest ones), but if that's what you decided to do for this project, that's fine by me